app/webpacker/styles/lead_providers.scss

```diff
@@ -0,0 +1,537 @@
+  top: 6px;
+}
+
+.index-group {
+  background: #f5f5f5;
+  padding: 1.5em;
+  margin-bottom: 1.5em;
+}
+.index-group p.govuk-body {
+  margin-bottom: 0;
+}
+.index-group .govuk-heading-m {
+  margin-bottom: 10px
+}
+.side-options {
+  background: transparent;
+  border: 1px solid #CCC;
+}
+.side-options .govuk-fieldset__legend--m {
+  margin-bottom: 0;
+  margin-top: 0;
+}
+.side-options .govuk-form-group {
+  margin-bottom: 15px;
+}
+.side-options .govuk-button {
+  margin-bottom: 10px;
+}
+
+.dashboard-numbers {
+  background: #f3f2f1;
+  padding: 1.5em;
+  margin-bottom: 1.5em;
+}
+.dashboard-numbers p {
+  margin-bottom: 0;
+  padding: 0;
+}
+.dashboard-numbers.key-number {
+  background: #1d70b8;
+}
+.dashboard-numbers.key-number p {
+  color: #FFF;
+}
+
+
+.app-navigation {
+  font-family: "GDS Transport",arial,sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 700;
+  font-size: 16px;
+  font-size: 1rem;
+  line-height: 1.25;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  width: 100%;
+  background-color: #f8f8f8;
+}
+.app-navigation {
+  font-size: 19px;
+  font-size: 1.1875rem;
+  line-height: 1.31579;
+  margin-top: -1px;
+  z-index: 10;
+  background: #f3f2f1;
+}
+.app-navigation:before {
+  position: absolute;
+  width: 100%;
+  background: #f3f2f1;
+  top: 105px;
+  left: 0;
+  height: 50px;
+  content: "";
+}
+.app-width-container {
+  max-width: 1100px;
+  margin: 0 auto;
+  position: relative;
+  z-index: 10;
+}
+.app-navigation__list {
+  position: relative;
+  left: -15px;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+.app-navigation__list-item {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  display: block;
+  position: relative;
+  height: 50px;
+  padding: 0 15px;
+  float: left;
+  line-height: 50px;
+}
+
+.app-navigation__link {
+  font-weight: 700;
+  text-decoration: none;
+}
+.govuk-link--no-visited-state:link {
+  color: #1d70b8;
+}
+.app-navigation__link:after {
+  content: "";
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+.app-navigation__list-item--current {
+  border-bottom: 4px solid #1d70b8;
+}
+
+form.top-right {
+  float: right;
+}
+
+
+@media (min-width: 48.0625em) {
+  .hmcts-pagination {
+    margin-left: -5px;
+    margin-right: -5px;
+    font-size: 0;
+    text-align: justify;
+  }
+  .hmcts-pagination:after {
+    content: '';
+    display: inline-block;
+    width: 100%;
+  }
+}
+
+.hmcts-pagination__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+@media (min-width: 48.0625em) {
+  .hmcts-pagination__list {
+    display: inline-block;
+    margin-bottom: 0;
+    vertical-align: middle;
+  }
+}
+
+@media (min-width: 48.0625em) {
+  .hmcts-pagination__results {
+    display: inline-block;
+    margin-bottom: 0;
+    vertical-align: middle;
+  }
+}
+
+.hmcts-pagination__item {
+  font-family: "GDS Transport", Arial, sans-serif;
```

Comment:
What? :D There is a "GDS Transport" font? :O

Reply:
That's the stock gov design font. I'd be surprised if we needed to specify that here. Although we might not be allowed to use it anyway: https://designnotes.blog.gov.uk/2015/03/11/can-i-use-the-gov-uk-fonts/

Reply:
To all above :) Relax - I'll clean it up

Reply:
Deleted anything not used